### DISCUSSION TITLE
不知道gpt总结怎么用

### DISCUSSION THREAD

**Discussion by agczsz** [DIYgod/RSSHub#15705] [Q&A]
不得不说，wiki给得非常潦草，并且在OPENAI_PROMPT还没给文章内容的参数，比较抽象
![image](https://github.com/DIYgod/RSSHub/assets/121206519/f4b90ee5-b1d3-4ac4-82e6-38976de50f33)
![image](https://github.com/DIYgod/RSSHub/assets/121206519/56053089-1aa2-4d96-b5c1-5e27fc2a5401)

**Comment by agczsz** [comment]
这，配置了一点输出没有：
![image](https://github.com/DIYgod/RSSHub/assets/121206519/cb14e014-b9bc-4959-b28a-6bdfdfae3969)

**Comment by Whichbfj28** [comment]
![G4 9$MZE6 R77P1T){XUCW4](https://github.com/user-attachments/assets/b4a5a98b-fe4a-40e6-b5c9-bfb63aa6eb3d)
配置图呢。
